### PR TITLE
OSDOCS-9662 Clean up 4.15 RN file

### PR DESCRIPTION
**Pull request by cbippley**
Version(s): 4.15

Issue: [OSDOCS-9662](https://issues.redhat.com/browse/OSDOCS-9662)

Link to docs preview: [4.15 Release Notes File](https://72080--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes)

QE review:
Not needed

Additional information:

### CODE REVIEW

**Comment by opayne1** [comment]
lines 1-22
- line 11 replace overhead with something like "running large queries can increase processor usage"
- line 22 s/as well as/and

**Comment by opayne1** [comment]
Line 76& 81 use Microsoft Azure as opposed to just Azure

**Comment by opayne1** [comment]
line 90 s/as well as/and

**Comment by opayne1** [comment]
line 104 is it supposed to be postinstallation? or post-installation? Vale says its postinstallation but I dont see it in the ssg or isg.

**Comment by opayne1** [comment]
Line 174 s/recomended/recommended

**Comment by opayne1** [comment]
line 282 extra space before "for more information"

**Comment by opayne1** [comment]
Line 591 extra space before "in this scenario" and extra address in the sentence

**Comment by opayne1** [comment]
680 and 690 s/prior to/before

**Comment by opayne1** [comment]
743 s/a lot of/many

**Comment by opayne1** [comment]
like 800 WDYT of migration instead of transition

**Comment by opayne1** [comment]
line 801 Kubernetes is misspelled

**Comment by opayne1** [comment]
line 839 "==== Log linking is enabled dby efault" should be "by default"

**Comment by opayne1** [comment]
line 907 s/in order to/to

**Comment by opayne1** [comment]
922 s/rolling stream/Rolling Stream

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -796,18 +796,18 @@ Support for version ranges::
...
-The Catalog API now uses an HTTP service to serve catalog content on the cluster. Previously, custom resource definitions (CRDs) were used for this purpose. The transition to using an HTTP service to serve catalog content reduces the load on the Kubenertes API server. For more information, see xref:../operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc#olmv1-finding-operators-to-install_olmv1-installing-operator[Finding Operators to install from a catalog].
+The Catalog API now uses an HTTP service to serve catalog content on the cluster. Previously, custom resource definitions (CRDs) were used for this purpose. The migration to using an HTTP service to serve catalog content reduces the load on the Kubernetes API server. For more information, see xref:../operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc#olmv1-finding-operators-to-install_olmv1-installing-operator[Finding Operators to install from a catalog].
 
 include::snippets/olmv1-cli-only.adoc[]
 
 For more information, see xref:../operators/olm_v1/index.adoc#olmv1-about[About Operator Lifecycle Manager 1.0].
+//
+//[id="ocp-4-15-osdk"]
+//=== Operator development
```
Comment (adellape) [inline release_notes/ocp-4-15-release-notes.adoc:806]: Sorry, my recently merged https://github.com/openshift/openshift-docs/pull/72071/files#diff-a4394e49205b55005aa75bf885c9f0d88e144344ff0c87ebe59d856269313337R826-R834 probably caused the rebase issue here 🙃

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -918,7 +918,7 @@ If the Cluster Monitoring Operator (CMO) reports task failures, the following re
 
 [id="ocp-4-15-network-observability-1-5"]
 === Network Observability Operator
-The Network Observability Operator releases updates independently from the {product-title} minor version release stream. Updates are available through a single, rolling stream which is supported on all currently supported versions of {product-title} 4. Information regarding new features, enhancements, and bug fixes for the Network Observability Operator is found in the xref:../network_observability/network-observability-operator-release-notes.adoc[Network Observability release notes].
+The Network Observability Operator releases updates independently from the {product-title} minor version release stream. Updates are available through a single, Rolling Stream which is supported on all currently supported versions of {product-title} 4. Information regarding new features, enhancements, and bug fixes for the Network Observability Operator is found in the xref:../network_observability/network-observability-operator-release-notes.adoc[Network Observability release notes].
```
Comment (adellape) [inline release_notes/ocp-4-15-release-notes.adoc:921]: Could use an anchor on the end of `.adoc`

File: release_notes/ocp-4-15-release-notes.adoc [hunk 3]
```diff
@@ -577,17 +576,17 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 |Supported
 |Supported
 |====
-
-[id="ocp-4-15-auth"]
-=== Authentication and authorization
+//
+//[id="ocp-4-15-auth"]
+//=== Authentication and authorization
```
Comment (adellape) [inline release_notes/ocp-4-15-release-notes.adoc:581]: Also my fault re: https://github.com/openshift/openshift-docs/pull/72071/files#diff-a4394e49205b55005aa75bf885c9f0d88e144344ff0c87ebe59d856269313337R826-R834 😬

**Pull request merged**